app/src/main/java/com/gathering/android/event/myevent/addevent/AddEventBottomSheetFragment.kt

```diff
@@ -31,9 +33,16 @@ class AddEventBottomSheetFragment : BottomSheetDialogFragment() {
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
-    ): View? {
+    ): View {
         binding = BottomSheetAddEventBinding.inflate(LayoutInflater.from(requireContext()))
 
+        val selectedImagePath = arguments?.getString(KEY_ARGUMENT_SELECTED_IMAGE)
+
+
+        if(selectedImagePath != null) {
```

Comment:
Not a big deal, but
This file needs to be reformatted, look at the `if(` it needs a space that shows this file is not formatted.